modules/ROOT/pages/securing.adoc

```diff
@@ -3,12 +3,66 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :keywords: security, securing
-:page-aliases: mule-security.adoc
-
-* xref:secure-configuration-properties.adoc[Secure Configuration Properties]
-* xref:cryptography.adoc[Cryptography Module]
- ** xref:cryptography-pgp.adoc[PGP]
- ** xref:cryptography-xml.adoc[XML]
- ** xref:cryptography-jce.adoc[JCE]
- ** xref:cryptography-reference.adoc[Crypto Module Documentation Reference]
-* xref:fips-140-2-compliance-support.adoc[FIPS 140-2 Compliance Support]
+:page-aliases: mule-security.adoc, configuring-security.adoc
+
+Security is an important aspect that you must take into account for your applications. +
+
+Businesses must ensure that the valuable information they store and make available through software applications and Web services is secure. Locked away and protected from unauthorized users and malicious attackers, protected resources, such as credit card information or Social Security numbers, must still be accessible to authorized legitimate users and systems in order to conduct business transactions. +
+
+To provide secure access to information, applications and services can apply a variety of security measures. Mule runtime engine provides several tools and methods that enable developers to protect applications according to security requirements, prevent security breaches and facilitate authorized access to data.
+Going from securing your application configuration properties to using cryptography capabilities, configuring FIPS 140-2 certified environment, securing flows with Spring Security, configuring TLS cryptographic protocol, obtaining access to protected resource using Oath Authorization Grant Types and configuring the Mule Secure Token Service.
+
+
+== Secure Configuration Properties
+Encrypt configuration properties for your applications. The process involves creating a secure configuration properties file, defining the secure properties in the file and configuring the file in your project with the Mule Secure Configuration Properties Extension module. +
+
+See details in xref:secure-configuration-properties.adoc[Secure Configuration Properties]
+
+== Cryptography Module
+The module provides cryptography capabilities to a Mule application. Main capabilities include:
+
+* Symmetric encryption and decryption of messages.
+* Asymmetric encryption and decryption of messages.
+* Message signing and signature validation of signed messages. +
+
+This module supports three different strategies to encrypt and sign your messages:
+
+* xref:cryptography-pgp.adoc[PGP] +
+ For signature/encryption.
+* xref:cryptography-xml.adoc[XML] +
+ For signature/encryption of XML documents or elements.
+* xref:cryptography-jce.adoc[JCE] +
+ For using a wider range of cryptography capabilities provided by the Java Cryptography Extension. +
+
+See details in xref:cryptography.adoc[Cryptography Module]
+
+== FIPS 140-2 Compliance Support
```

Comment:
However you decide to word the introductory list of tools and methods, make sure that the wording of the listed items match the wording of the respective subtopics.

"compliance support" sounds a bit redundant.